src/cpp/src/streamer_base.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/genai/streamer_base.hpp"
+
+namespace ov {
+namespace genai {
+
+ov::genai::StreamerBase::~StreamerBase() = default;
```

Comment:
maybe we can define it in `text_callback_streamer.cpp` (or rename this file to be more generic) to avoid new files with a single line of code